### ISSUE TITLE
When enabling BPTC format in a project, Godot doesn't pick the BC4 compressing format for normal maps automatically anymore.

### ISSUE THREAD

**Issue by Stof25** [godotengine/godot#38927]
**Godot version:**
v3.2.1.stable.official

**OS/device including version:**
Windows 10

**Issue description:**
When you enable support for BC7 (BTPC) formats in the project options and set some normal map texture to the VRAM compression option, Godot will pick by default BC7 a 3 channel compression format instead of the usual BC4 high quality two channel format. This is because the "Bptc Ldr" option is set to Enabled by default and seems to take precedence in the compression format decision over the "Normal Map : Enabled" setting.

**Steps to reproduce:**
From a new project, enable the Bptc compression setting. Add a normal map texture. Set it's import options to VRAM and plug it in a material as the normal map.

**Minimal reproduction project:**
[normalBC7Bug.zip](https://github.com/godotengine/godot/files/4663336/normalBC7Bug.zip)

**Work around:**
Manually disable BC7 for all Normal Map textures by setting the "Bptc Ldr" import option to "RGBA Only"

**Comment by Calinou** [comment]
Did it work in a previous Godot version?

**Comment by Stof25** [comment]
I have no idea really, it's the first time I used the BTPC option in the first place. I suspect it's just never worked since it was implemented as such. It seems mostly light an oversight in the compression format decision heuristics.

```cpp
		if (is_ldr && can_bptc) {
			if (bptc_ldr == 0 || (bptc_ldr == 1 && !(used_channels == Image::USED_CHANNELS_LA || used_channels == Image::USED_CHANNELS_RGBA))) {
				can_bptc = false;
			}
		}
```
https://github.com/godotengine/godot/blob/00949f0c5fcc6a4f8382a4a97d5591fd9ec380f8/editor/import/resource_importer_layered_texture.cpp#L346

bptc/BC7 blacklists specifically the RGBA or LA texture formats. BC7 probably shouldn't also ignore RG. Or only apply to RGB really.
